samples/language/Comments.qs

```diff
@@ -17,4 +17,4 @@ namespace MyQuantumApp {
         // preface what they refer to.
         return [];
     }
-}
```

Comment:
There doesn't appear to be any change in this file other than the end of file, can you please remove this file from the PR?

Reply:
Yes, my editor keeps adding those. I thinks its a prettier flaw